modules/cluster-logging-collector-log-forward-cloudwatch.adoc

```diff
@@ -0,0 +1,294 @@
+// Module included in the following assemblies:
+//
+// * logging/cluster-logging-external.adoc
+
+[id="cluster-logging-collector-log-forward-cloudwatch_{context}"]
+= Forwarding logs to Amazon CloudWatch
+
+You can forward logs to Amazon CloudWatch, a hosted monitoring and log storage service. You can forward logs to CloudWatch in addition to, or instead of, the default OpenShift Logging-managed Elasticsearch log store.
+
+To configure log forwarding to CloudWatch, create a `ClusterLogForwarder` custom resource (CR) with an output for CloudWatch and a pipeline that uses the output.
+
+.Procedure
+
+. Create a `Secret` YAML file similar to the following example. Use base64-encoded values for the `aws_access_key_id` and `aws_secret_access_key` fields. For example:
++
+[source,yaml]
+----
+apiVersion: v1
+kind: Secret
+metadata:
+  name: cw-secret
+  namespace: openshift-logging
+data:
+  aws_access_key_id: QUtJQUlPU0ZPRE5ON0VYQU1QTEUK
+  aws_secret_access_key: d0phbHJYVXRuRkVNSS9LN01ERU5HL2JQeFJmaUNZRVhBTVBMRUtFWQo=
+----
+
+
+. Create the secret. For example:
++
+[source,terminal]
+----
+$ oc apply -f cw-secret.yaml
+----
+
+
+. Create a `ClusterLogForwarder` CR YAML file. In it, specify the name of the secret. For example:
++
+[source,yaml]
+----
+apiVersion: "logging.openshift.io/v1"
+kind: ClusterLogForwarder
+metadata:
+  name: instance <1>
+  namespace: openshift-logging <2>
+spec:
+  outputs:
+   - name: cw <3>
+     type: cloudwatch <4>
+     cloudwatch:
+       groupBy: logType <5>
+       region: us-east-2 <6>
+     secret:
+        name: cw-secret <7>
+  pipelines:
+    - name: infra-logs <8>
+      inputRefs: <9>
+        - infrastructure
+        - audit
+        - application
+      outputRefs:
+        - cw <10>
+----
+<1> The name of the `ClusterLogForwarder` CR must be `instance`.
+<2> The namespace for the `ClusterLogForwarder` CR must be `openshift-logging`.
+<3> Specify a name for the output.
+<4> Specify the `cloudwatch` type.
+<5> Optional: Specify the option CloudWatch uses to group the logs. If you omit `groupBy`, its default value is `logType`.
+<6> Specify the region your CloudWatch instance uses.
```

Comment:
This sounds a bit off. CloudWatch is like GMail, there's just a single instance of it. May be just say "Specify the AWS region to connect to"?

Reply:
Updated.